src/applications/mhv-medical-records/tests/fixtures/chemHem2.json

```diff
@@ -0,0 +1,317 @@
+{
+  "resourceType" : "DiagnosticReport",
+  "id" : "9953",
+  "meta" : {
+    "versionId" : "22",
+    "lastUpdated" : "2024-05-16T18:08:59.215-04:00",
+    "source" : "#reWo6V0WdeyFTR19",
+    "profile" : [
+      "https://department-of-veterans-affairs.github.io/mhv-fhir-phr-mapping/StructureDefinition/VA.MHV.PHR.chReport"
+    ]
+  },
+  "text" : {
+    "status" : "generated",
+    "div" : "<div xmlns=\"http://www.w3.org/1999/xhtml\"><h2><span title=\"Codes: \">CH</span> (<span title=\"Codes: {http://terminology.hl7.org/CodeSystem/v2-0074 LAB}\">Laboratory</span>, <span title=\"Codes: {http://terminology.hl7.org/CodeSystem/v2-0074 CH}\">Chemistry</span>, <span title=\"Codes: {http://loinc.org 2345-7}\">GLUCOSE:MCNC:PT:SER/PLAS:QN:</span>) </h2><table class=\"grid\"><tr><td>Subject</td><td><b>L MHVSYSTEST </b> unknown, DoB: 1000-01-01 ( <code>urn:oid:2.16.840.1.113883.4.349</code>/942104\u00a0(use:\u00a0usual))</td></tr><tr><td>When For</td><td>2017-08-02 09:50:57-0400</td></tr><tr><td>Reported</td><td>2017-08-02 09:52:27-0400</td></tr><tr><td>Identifier:</td><td> <code>urn:fdc:TEST.DAYTON.MED.VA.GOV:LR</code>/1172140001\u00a0(use:\u00a0usual)</td></tr></table><p><b>Report Details</b></p><table class=\"grid\"><tr><td><b>Code</b></td><td><b>Value</b></td><td><b>Reference Range</b></td><td><b>Flags</b></td><td><b>Note</b></td><td><b>When For</b></td></tr><tr><td colspan=\"6\"><i>This Observation could not be resolved</i></td></tr></table></div>"
+  },
+  "contained" : [
+    {
+      "resourceType" : "Specimen",
+      "id" : "Specimen-0",
+      "meta" : {
+        "profile" : [
+          "https://department-of-veterans-affairs.github.io/mhv-fhir-phr-mapping/StructureDefinition/VA.MHV.PHR.chSpecimen"
+        ]
+      },
+      "status" : "available",
+      "type" : {
+        "coding" : [
+          {
+            "system" : "http://terminology.hl7.org/CodeSystem/v2-0487",
+            "code" : "SER",
+            "display" : "Serum"
+          },
+          {
+            "system" : "http://va.gov/terminology/vistaDefinedTerms/61",
+            "version" : "5.2",
+            "code" : "72",
+            "display" : "SERUM"
+          }
+        ],
+        "text" : "SERUM"
+      },
+      "request" : [
+        {
+          "reference" : "#ServiceRequest-1"
+        }
+      ],
+      "collection" : {
+        "collectedDateTime" : "2017-08-02T09:50:57-04:00"
+      }
+    },
+    {
+      "resourceType" : "Practitioner",
+      "id" : "Provider-1",
+      "identifier" : [
+        {
+          "system" : "http://va.gov/terminology/vistaDefinedTerms/4",
+          "value" : "14934-VA552"
+        }
+      ],
+      "name" : [
+        {
+          "family" : "HALL",
+          "given" : [
+            "LISA",
+            "A"
+          ]
```

Comment:
Can you change this to "Jane A Doe" or something similarly generic? Same if there are any other names in here I missed.

Reply:
Yes!